understanding/20/low-or-no-background-audio.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Low or No Background Audio</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Prerecorded speech is not disrupted by background sound</dd>
-         <dt>Author task</dt><dd>Avoid or lessen background sound, or let users turn it off</dd>
+         <dt>Goal</dt><dd>Prerecorded speech is not disrupted by background sound.</dd>
+         <dt>What to do</dt><dd>Avoid or lessen background sound, or let users turn it off.</dd>
+         <dt>Why it's important</dt><dd>People who are hard of hearing may have difficulty distinguishing speech from music and other sounds.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>People who are hard of hearing may have difficulty understanding what is said when there is music and other sounds in background unless much quieter than speech.</dd> 
```

Reply:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>People who are hard of hearing may have understanding speech when there is music or other background sounds that are not much quieter.</dd> 
```

Reply:
I don't find either of the wordier suggestions add much clarity to this brief concept; and undeniably, wordier is less brief, therefore counterproductive.

Reply:
I also feel the shorter version is more complete. The longer version points to background sounds being much quieter. On one hand, it lacks a definition of what 'much quieter' is (and there is no common understanding), on the other, it does not include to avoid background music completely.